dsb-gui/src/main/resources/lang_en.properties

```diff
@@ -31,13 +31,18 @@ ds3NewBucketToolTip=New Bucket
 ds3DeleteButtonToolTip=Delete Objects
 ds3TransferLeftToolTip=Transfer to local system
 newSessionButton=New Session
+unableToShowVersionPresenter=Unable to show Version Presenter
+versionView=Versions
+unableToGetVersions=Unable to get file versions
+couldNotGetVersion=Could not get Versions
```

Comment:
whats the difference between unableToGetVersions and couldNotGetVersion?

Reply:
I was thinking one is more generic?

Reply:
Which one is more generic though?